security/advancedtls/advancedtls_test.go

```diff
@@ -679,3 +681,83 @@ func TestOptionsConfig(t *testing.T) {
 		})
 	}
 }
+
+func TestGetCertificateSNI(t *testing.T) {
+	// Load server certificates for setting the serverGetCert callback function.
+	serverPeerCert1, err := tls.LoadX509KeyPair(testdata.Path("server_cert_1.pem"),
+		testdata.Path("server_key_1.pem"))
+	if err != nil {
+		t.Fatalf("Server is unable to parse peer certificates. Error: %v", err)
+	}
+	serverPeerCert2, err := tls.LoadX509KeyPair(testdata.Path("server_cert_2.pem"),
+		testdata.Path("server_key_2.pem"))
+	if err != nil {
+		t.Fatalf("Server is unable to parse peer certificates. Error: %v", err)
+	}
+	serverPeerCert3, err := tls.LoadX509KeyPair(testdata.Path("server_cert_3.pem"),
+		testdata.Path("server_key_3.pem"))
+	if err != nil {
+		t.Fatalf("Server is unable to parse peer certificates. Error: %v", err)
+	}
+	tests := []struct {
+		desc                string
+		serverGetCert       func(*tls.ClientHelloInfo) ([]*tls.Certificate, error)
+		serverName          string
+		expectedCertificate tls.Certificate
```

Comment:
s/expectedCertificate/wantCert.

`want` is preferred over `expected` in identifier names in tests since it is shorter and conveys the same meaning.

Reply:
Ok!